<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите новое имя товара</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите цену товара</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите количество товара</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="take_photo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: #ffdbcc;
}

QPushButton:pressed {
	background-color: #fcb9aa;
}
QPushButton {
	background-color: rgb(57, 208, 208);
	border-radius: 31px;
}</string>
       </property>
       <property name="text">
        <string>Выбрать картинку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color: #ffdbcc;
}

QPushButton:pressed {
	background-color: #fcb9aa;
}
QPushButton {
	background-color: rgb(57, 208, 208);
	border-radius: 31px;
}</string>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
